package-lock.json

```diff
@@ -2809,7 +2809,6 @@
       "resolved": "https://registry.npmjs.org/@styled-system/background/-/background-5.1.2.tgz",
       "integrity": "sha512-jtwH2C/U6ssuGSvwTN3ri/IyjdHb8W9X/g8Y0JLcrH02G+BW3OS8kZdHphF1/YyRklnrKrBT2ngwGUK6aqqV3A==",
       "dev": true,
-      "optional": true,
```

Comment:
Not sure why there are changes here, please discard your changes to this file

Reply:
sure.